HTML_CSS_Js/exercise-05/frameScript.js

```diff
@@ -0,0 +1,78 @@
+let Data=[]
+function fetchData(){
+console.log(document.URL)
+var id =document.URL.substring(document.URL.lastIndexOf('=') + 1);
+fetch("https://pokeapi.co/api/v2/pokemon/"+id)
+.then(response => response.json())
+.then(function(pokeData){
+            Data.push(pokeData)
+            displayPokemon()
+        })
+        .catch( e=>{
+            console.log(e)
+        })
+    
+}
+function displayPokemon()
+{
+let pokeName=document.getElementById('name') 
+let pokeWeight=document.getElementById('weight') 
+let pokeExperience=document.getElementById('experience') 
+let pokeSpecies=document.getElementById('species') 
+let pokeStats=document.getElementById('stat') 
+let pokeForms=document.getElementById('forms') 
+let pokeMoves=document.getElementById('moves') 
+let pokeImage=document.getElementById('sprite')
+console.log(Data[0])
+pokeImage.src=`${Data[0].sprites.front_default}`
+pokeImage.alt="image loading ..."
```

Comment:
`alt` attribute should describe the image in case it doesn't load, so perhaps image of _pokemon name_